2020-04-07 22:02:14.219 [main] ERROR com.ranjit.o2o.service.impl.WechatAuthServiceImpl - insertWechatAuth error:org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dafahizhfdhaih' for key 'open_id'
### The error may exist in file [D:\IDEA envirment\o2o\target\classes\mapper\WechatAuthDao.xml]
### The error may involve com.ranjit.o2o.dao.WechatAuthDao.insertWechatAuth-Inline
### The error occurred while setting parameters
### SQL: INSERT   INTO   tb_wechat_auth(user_id,open_id,create_time)   VALUES   (?,?,?)
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dafahizhfdhaih' for key 'open_id'
; Duplicate entry 'dafahizhfdhaih' for key 'open_id'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dafahizhfdhaih' for key 'open_id'
2020-04-07 22:07:02.270 [main] ERROR com.ranjit.o2o.service.impl.WechatAuthServiceImpl - insertWechatAuth error:org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dafahizhfdhaih' for key 'open_id'
### The error may exist in file [D:\IDEA envirment\o2o\target\classes\mapper\WechatAuthDao.xml]
### The error may involve com.ranjit.o2o.dao.WechatAuthDao.insertWechatAuth-Inline
### The error occurred while setting parameters
### SQL: INSERT   INTO   tb_wechat_auth(user_id,open_id,create_time)   VALUES   (?,?,?)
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dafahizhfdhaih' for key 'open_id'
; Duplicate entry 'dafahizhfdhaih' for key 'open_id'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry 'dafahizhfdhaih' for key 'open_id'
